src/internal/m365/collection/groups/export.go

```diff
@@ -26,7 +31,13 @@ func NewExportCollection(
 	backupVersion int,
 	cec control.ExportConfig,
 	stats *metrics.ExportStats,
+	cat path.CategoryType,
 ) export.Collectioner {
+	streamItems := streamChannelMessages
+	if cat == path.ConversationPostsCategory {
+		streamItems = streamConversationPosts
+	}
```

Comment:
might want a switch with channels, posts, default (err) isntead of this pattern, since groups can handle more categories than these two.  Worst case scenario it could prevent or identify an accidental usage of this func.